src/mount/mount-tool.c

```diff
@@ -1104,27 +1104,40 @@ static int acquire_removable(struct udev_device *d) {
         return 1;
 }
 
+static int discover_loop_backing_file(void) {
+        _cleanup_free_ char *escaped = NULL;
+
+        if (arg_mount_where)
+                return 0;
+
+        escaped = xescape(basename(arg_mount_what), "\\");
```

Comment:
OK. I've added oom check here in the revised version.

Here, the argument is `arg_mount_what`, not `arg_mount_where`.